code/Controllers/CMSMain.php

```diff
@@ -1071,7 +1071,7 @@ public function Breadcrumbs($unlinked = false)
         $record = $this->currentPage();
         if (!$record) {
             $items->push(new ArrayData(array(
-                'Title' => CMSPagesController::menu_title(),
+                'Title' => $this->menu_title(CMSPagesController::class),
```

Comment:
doesn't this involve turning the static method into a non-static method?

Reply:
I think the parent method `menu_title` doesn't need to be a static method as it expects the $class parameter; https://github.com/silverstripe/silverstripe-admin/blob/1/code/LeftAndMain.php#L896-L899

Or are you referring to something else?

Reply:
Whether it needs to be static or not is a bit irrelevant because the function is static and that can't change until SS5.

As the docs say the first param is for use only when calling directly from `LeftAndMain` (`@param string $class Optional class name if called on LeftAndMain directly`)